api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirConceptDao.java

```diff
@@ -11,11 +11,13 @@
 
 import java.util.Optional;
 
+import javax.validation.constraints.NotNull;
+
 import org.openmrs.Concept;
 
-public interface FhirConceptDao {
+public interface FhirConceptDao extends FhirDao<Concept> {
```

Comment:
@rnjn bacher hi ian,i got realise that we don't have a model for concept under org.hl7.fhir.r4.model,any thoughts for that since this is needed around here public interface FhirConceptDao extends FhirDao<Concept>

Reply:
@HerbertYiga I think OpenMRS `Concept` maps to `CodeableConcept` in Fhir. Hope that helps. 

cc @ibacher

Reply:
@HerbertYiga: It's exactly what @VaishSiddharth said. We aren't actually publishing any FHIR-related concept resources. We're just using `CodeableConcept` in a few places where it's required.